src/pages/Home.jsx

```diff
@@ -1,3 +1,36 @@
-const Home = () => <h1>Welcome to Voma</h1>;
+import styled from 'styled-components';
 
-export default Home;
+const StyledSection = styled.section`
+    width: 50%;
+    margin: 0 auto;
+    padding-top: 100px;
+    font-family: 'IBM Plex Sans', sans-serif;
+`;
+
+const StyledH1 = styled.h1`
+    font-size: 48px;
+    font-weight: 600;
+`;
+
+const StyledParagraph = styled.p`
+    font-size: 18px;
+`;
+
+const StyledImg = styled.img`
+    width: 50%;
+`;
+
+const StyledA = styled.a`
+    color: blue;
+    text-decoration: underline;
```

Comment:
Seems like this <a> styles are used everywhere. Can we add them to the Global Styles instead?

Reply:
added